packages/messaging/src/models/token-details-model.ts

```diff
@@ -39,29 +41,31 @@ ValidateInput.prototype.fcmPushSet;
 
 export default class TokenDetailsModel extends DBInterface {
   constructor() {
-    super(TokenDetailsModel.DB_NAME, DB_VERSION);
+    super(DB_NAME, DB_VERSION);
   }
 
-  static get DB_NAME() {
-    return 'fcm_token_details_db';
-  }
+  onDBUpgrade(db: IDBDatabase, evt: IDBVersionChangeEvent) {
+    if (evt.oldVersion < 1) {
```

Comment:
this means it's brand new, right? can you add a quick comment?

Reply:
Added comment